core/input/input_event.cpp

```diff
@@ -132,6 +132,9 @@ void InputEvent::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("xformed_by", "xform", "local_ofs"), &InputEvent::xformed_by, DEFVAL(Vector2()));
 
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "device"), "set_device", "get_device");
+
+	BIND_CONSTANT(DEVICE_ID_EMULATION);
+	BIND_CONSTANT(DEVICE_ID_INTERNAL);
```

Comment:
I suggest to not expose `DEVICE_ID_INTERNAL`, because the user will never receive any input event in GDScript with this device id.